types/loadable__server/index.d.ts

```diff
@@ -51,32 +51,32 @@ export class ChunkExtractor {
 	/**
 	 * Get scripts as a string of `<script>` tags
 	 */
-	getScriptTags(): string;
+	getScriptTags(attributes?: { [key: string]: string }): string;
```

Comment:
It looks like it not only can accept an object, but also a function as documented here:
https://www.smooth-code.com/open-source/loadable-components/docs/api-loadable-server/#chunkextractorgetscripttags

Also, specifying that it's of type `{ [key: string]: string }` seems incorrect. I don't see where it enforces that property value should be a `string`. I think this can be just `object`.

Reply:
Hey @martynaskadisa, thanks for reviewing the PR.

> It looks like it not only can accept an object, but also a function as documented here

Yeah, I saw that but the fact is... the docs are wrong, the code is not there. Those functions don't accept an `attrFn` yet. For that reason I didn't add it. Maybe we should ping them to explain.

> Also, specifying that it's of type { [key: string]: string } seems incorrect. I don't see where it enforces that property value should be a string. I think this can be just object.

Well, those attributes are going to end up being strings in the dom, so I guess it is good to have them as strings in typescript.

cc: @jsfez @neoziro @rhy0u could you please take a look at these two issues?

Reply:
> Yeah, I saw that but the fact is... the docs are wrong, the code is not there. Those functions don't accept an attrFn yet. For that reason I didn't add it. Maybe we should ping them to explain.

Ahh, I see. It seems this behaviour has only landed in master and has not been released yet https://github.com/smooth-code/loadable-components/pull/277.

Reply:
Oh, good catch! Let's wait until they release it then.

Regarding the other one, let's see if they can chime in here as well.

Reply:
https://github.com/smooth-code/loadable-components/releases/tag/v5.8.0

I trust you guys for typings!

Reply:
Ok, thanks @neoziro!

@martynaskadisa I've made another push with `{}` instead of `{ [key: string]: string }` and `attrFn`.

Reply:
@luisherranz Awesome!